frontend/package.json

```diff
@@ -13,7 +13,7 @@
     "build": "vite build",
     "lint": "eslint src --max-warnings 0",
     "start": "vite",
-    "start:heroku": "UNLEASH_API=https://unleash.herokuapp.com yarn run start",
+    "start:heroku": "UNLEASH_API=https://unleash.herokuapp.com yarn run start", // <— what do we here?
```

Comment:
Will this continue to work?

Reply:
Technically it should work, but I'm guessing we don't want to keep them running, instead possibly update links to the new sandbox instances @gardleopard has set up.

Reply:
I'd be happy to do that, yeah! @gardleopard, do you have any preference on which  targets to use?

Reply:
sanbox.getunleash.ai/ospro